cinema/migrations/0001_initial.py

```diff
@@ -73,6 +76,9 @@
                 ),
                 ("title", models.CharField(max_length=255)),
                 ("description", models.TextField()),
+                ("duration", models.IntegerField()),
+                ("actors", models.ManyToManyField(to="cinema.actor")),
+                ("genres", models.ManyToManyField(to="cinema.genre")),
```

Comment:
The `genres` field in the Movie model uses a ManyToManyField to the Genre model, allowing a movie to belong to multiple genres. This is correctly implemented.